src/utils/rest-api.js

```diff
@@ -774,11 +774,15 @@ export function fetchNetworkModificationTree(studyUuid) {
     console.info('Fetching network modification tree');
     const url = getStudyUrl(studyUuid) + '/tree';
     console.debug(url);
-    return backendFetch(url, { method: 'get' }).then((response) =>
-        response.ok
-            ? response.json()
-            : response.text().then((text) => Promise.reject(text))
-    );
+    return backendFetch(url, { method: 'get' }).then((response) => {
+        if (response.ok) {
+            return response.json();
+        } else {
+            return response.json().then((text) => {
+                return Promise.reject(text);
+            });
+        }
+    });
```

Comment:
why this change ? It's exactly the same or i'm misssing something ?

Reply:
Not exactly. For "exception path", content was awaited as flat string and is in fact json, as seen in browser console. This json is to be exploited in the caller.